--- a/cmake/SFMLConfigDependencies.cmake.in
+++ b/cmake/SFMLConfigDependencies.cmake.in
@@ -10,6 +10,8 @@ if(SFML_STATIC_LIBRARIES)
         set(FIND_SFML_OS_WINDOWS 1)
     elseif(${CMAKE_SYSTEM_NAME} MATCHES "Android")
         set(FIND_SFML_OS_ANDROID 1)
+    elseif(${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
+        set(FIND_SFML_OS_IOS 1)
     elseif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
         set(FIND_SFML_OS_LINUX 1)
