--- a/src/SFML/Graphics/Image.cpp
+++ b/src/SFML/Graphics/Image.cpp
@@ -120,16 +120,7 @@ void Image::create(unsigned int width, u
 ////////////////////////////////////////////////////////////
 bool Image::loadFromFile(const std::string& filename)
 {
-    #ifndef SFML_SYSTEM_ANDROID
-
-        return priv::ImageLoader::getInstance().loadImageFromFile(filename, m_pixels, m_size);
-
-    #else
-
-        priv::ResourceStream stream(filename);
-        return loadFromStream(stream);
-
-    #endif
+    return priv::ImageLoader::getInstance().loadImageFromFile(filename, m_pixels, m_size);
 }


--- a/src/SFML/Graphics/Font.cpp
+++ b/src/SFML/Graphics/Font.cpp
@@ -140,8 +140,6 @@ Font::~Font()
 ////////////////////////////////////////////////////////////
 bool Font::loadFromFile(const std::string& filename)
 {
-    #ifndef SFML_SYSTEM_ANDROID
-
     // Cleanup the previous resources
     cleanup();
     m_refCount = new int(1);
@@ -189,16 +187,6 @@ bool Font::loadFromFile(const std::strin
     m_info.family = face->family_name ? face->family_name : std::string();

     return true;
-
-    #else
-
-    if (m_stream)
-        delete static_cast<priv::ResourceStream*>(m_stream);
-
-    m_stream = new priv::ResourceStream(filename);
-    return loadFromStream(*static_cast<priv::ResourceStream*>(m_stream));
-
-    #endif
 }


