diff --git a/CMakeLists.txt b/CMakeLists.txt
index 40cfa1b..db8c147 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,7 +57,7 @@ else()
     if(SFML_OS_IOS)
         set(BUILD_SHARED_LIBS FALSE)
     elseif(SFML_OS_ANDROID)
-        set(BUILD_SHARED_LIBS TRUE)
+        set(BUILD_SHARED_LIBS FALSE)
     endif()
 endif()

diff --git a/src/SFML/Main/CMakeLists.txt b/src/SFML/Main/CMakeLists.txt
index 9390aa9..0180fb3 100644
--- a/src/SFML/Main/CMakeLists.txt
+++ b/src/SFML/Main/CMakeLists.txt
@@ -35,5 +35,5 @@ set_target_properties(sfml-main PROPERTIES
 # from depending on shared libraries), we need a boostrap activity which
 # will load our shared libraries manually
 if(SFML_OS_ANDROID)
-    sfml_add_library(sfml-activity SOURCES ${SRCROOT}/SFMLActivity.cpp)
+    sfml_add_library(sfml-activity STATIC SOURCES ${SRCROOT}/SFMLActivity.cpp)
 endif()
